addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_image.py

```diff
@@ -27,14 +27,16 @@
 from io_scene_gltf2.io.exp.gltf2_io_user_extensions import export_user_extensions
 
 
-@cached
+# @cached
```

Comment:
export_image_class appears to not be hashable which causes this to fail. Not ideal.